README.md

```diff
@@ -37,56 +36,81 @@ If you find a typo or you feel like you can improve the HTML, CSS, or JavaScript
+[terraform-docs-agents]: https://github.com/hashicorp/terraform-docs-agents
+[terraform-plugin-sdk]: https://github.com/hashicorp/terraform-plugin-sdk
+[terraform-plugin-log]: https://github.com/hashicorp/terraform-plugin-log
+[terraform-plugin-mux]: https://github.com/hashicorp/terraform-plugin-mux
+[terraform-plugin-framework]: https://github.com/hashicorp/terraform-plugin-framework
 
-- This repository, under `content/`:
-
-  - Terraform Cloud docs
-  - Terraform Enterprise docs
-  - Plugin Development
-  - Terraform Registry Publishing
 
   **Notable branches:** `master` is the "live" content that gets deployed to [terraform.io](https://terraform.io). The site gets redeployed for new commits to master.
 
-- [hashicorp/terraform](https://github.com/hashicorp/terraform), under `website/docs`:
-
-  - Terraform CLI docs
-  - Terraform Language docs
-
-  **Notable branches:** `stable-website` is the "live" content that gets deployed to [terraform.io](https://terraform.io), but docs changes should get merged to `main` (and/or one of the long-lived version branches) first. See [More About `stable-website`][#more-about-stable-website] below for more details.
-
-- [hashicorp/terraform-cdk](https://github.com/hashicorp/terraform-cdk), under `website/docs`:
-
-  - Terraform CDK docs
-
-  **Notable branches:** `stable-website` is the "live" content that gets deployed to [terraform.io](https://terraform.io), but docs changes should get merged to `main` first. See [More About `stable-website`][#more-about-stable-website] below for more details.
 
 ## Deploying Changes to [terraform.io](https://terraform.io)
 
 ### For changes in this repo
 
-Merge the PR to master, and the site will automatically deploy in about 20m. 🙌
-
-### For changes in `hashicorp/terraform`
-
-Merge the PR to `main`. The changes will appear in the next major Terraform release.
-
-If you need your changes to be deployed sooner, cherry-pick them to:
-
-- the current release branch (e.g. `v1.0`) and push. They will be deployed in the next minor version release (once every two weeks).
-- the `stable-website` branch and push. They will be included in the next site deploy (see below). Note that the release process resets `stable-website` to match the release tag, removing any additional commits. So, we recommend always cherry-picking to the version branch first and then to `stable-website` when needed.
+Merge the PR to master, and the site will automatically deploy in about 5m. 🙌
 
-Once your PR to `stable-website` is merged, open a PR bumping the submodule commit in `terraform-website`. See the [Using Submodules](#using-submodules) section below for more details.
+### For changes in any other listed repo
 
-#### Backport Tags
+There are various GitHub refs that will cause docs content to be deployed to terraform.io,
+and these may vary on a repo-to-repo basis but generally speaking, these are limited to `stable-website`, `main`, and a few others.
```

Comment:
If we're not going to mention the others, maybe just stick with how the latest docs are updated? `and a few others` isn't too helpful 😄